PubNub/Misc/PNPrivateStructures.h

```diff
@@ -12,19 +12,37 @@
 #define PNPrivateStructures_h
 
 /**
- @brief  Stores reference on key under which unique user identifier will be stored persistently.
+ * @brief Stores reference on key under which unique user identifier will be stored persistently.
  */
 extern NSString * const kPNConfigurationUUIDKey;
 
 /**
- @brief  Helper to stringify operation type in result and status objects.
+ * @brief Options describe object's message type.
+ *
+ * @since 4.9.0
+ */
+typedef NS_OPTIONS(NSUInteger, PNMessageType) {
+    /**
+     @brief Type which represent regular message object.
+     */
+    PNRegularMessageType = 0,
```

Comment:
A non-existent "e" is also a regular message.

Reply:
And this is why `regular` message type has been assigned **0**. When we pull out information from envelope and there is no value for requested property, we will get `nil`. Because code treat received value as number and try to get from it `NSUInteger` it will get **0** in response (if we tried to get `boolValue` - it will became **NO**).